src/ui/components/Panel/Panel.styles.js

```diff
@@ -0,0 +1,55 @@
+import { makeStyles } from "@material-ui/core";
+import mq from "../../../config/mq";
+const useStyles = makeStyles((theme) => ({
+  container: {
+    margin:10,
+    borderRadius:10,
+    backgroundColor:'#fff',
+    paddingTop:1,
+    width:'80%',
+    fontSize: 13,
+  },
+  tabla: {
+    margin: 10,
+    backgroundColor: "#fff",
```

Comment:
All colors must have to come from the main theme palette colorsm, something like this 
`theme.color.palette.colors `